pkg/v2/resource/chartconfig/resource.go

```diff
@@ -10,8 +10,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/giantswarm/cluster-operator/pkg/cluster"
-	"github.com/giantswarm/cluster-operator/pkg/v1/guestcluster"
-	"github.com/giantswarm/cluster-operator/pkg/v1/key"
+	"github.com/giantswarm/cluster-operator/pkg/v2/guestcluster"
+	"github.com/giantswarm/cluster-operator/pkg/v2/key"
```

Comment:
Below is the resource name which should be `chartconfigv2`. Lets make sure we have this sorted for all resources.

Reply:
Good catch. I applied the replace function from original script but apparently it didn't catch these. Will fix right away.